### ISSUE TITLE
Panic running hercules combine

### ISSUE THREAD

**Issue by erizocosmico** [src-d/hercules#215]
Panic running combine.

```
panic: runtime error: index out of range

goroutine 1 [running]:
gopkg.in/src-d/hercules.v7/leaves.(*CouplesAnalysis).MergeResults.func2(0xc000bca000, 0x261, 0x261, 0xc000bb2000, 0x261, 0x400, 0xc000a9e300, 0x1, 0x1)
	/home/travis/gopath/src/gopkg.in/src-d/hercules.v7/leaves/couples.go:340 +0x2c1
gopkg.in/src-d/hercules.v7/leaves.(*CouplesAnalysis).MergeResults(0xc00015c070, 0x12088e0, 0xc00004a300, 0x12088e0, 0xc00004a580, 0xc000b60030, 0xc000bd4420, 0x11b4500, 0xc000aaa940)
	/home/travis/gopath/src/gopkg.in/src-d/hercules.v7/leaves/couples.go:345 +0x84b
main.mergeResults(0xc000b49950, 0xc000b60030, 0xc000a8ce70, 0xc000bd4420)
	/home/travis/gopath/src/gopkg.in/src-d/hercules.v7/cmd/hercules/combine.go:157 +0x1b4
main.glob..func1(0x20c48e0, 0xc000032600, 0x259, 0x259)
	/home/travis/gopath/src/gopkg.in/src-d/hercules.v7/cmd/hercules/combine.go:42 +0x794
gopkg.in/src-d/hercules.v7/vendor/github.com/spf13/cobra.(*Command).execute(0x20c48e0, 0xc0000eac00, 0x259, 0x260, 0x20c48e0, 0xc0000eac00)
	/home/travis/gopath/src/gopkg.in/src-d/hercules.v7/vendor/github.com/spf13/cobra/command.go:766 +0x2cc
gopkg.in/src-d/hercules.v7/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x20c4da0, 0xc000a9e660, 0xc000a9e630, 0xc000476700)
	/home/travis/gopath/src/gopkg.in/src-d/hercules.v7/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
gopkg.in/src-d/hercules.v7/vendor/github.com/spf13/cobra.(*Command).Execute(0x20c4da0, 0x843150, 0xc0000be058)
	/home/travis/gopath/src/gopkg.in/src-d/hercules.v7/vendor/github.com/spf13/cobra/command.go:800 +0x2b
main.main()
	/home/travis/gopath/src/gopkg.in/src-d/hercules.v7/cmd/hercules/root.go:509 +0x31
ERROR:root:Aggregating of statistics failed with exception Command
```

**Comment by vmarkovtsev** [comment]
Can you please send me the list of files which led to this crash. As a workaround, use `--only` and at least combine those analyses which are combinable.

**Comment by vmarkovtsev** [comment]
Fixed in v8.3.2
